pom.xml

```diff
@@ -1582,7 +1584,8 @@
         <jdk>11</jdk>
       </activation>
       <properties>
-        <java.version>1.8</java.version>
+        <enforce.versions.skip>true</enforce.versions.skip>
```

Comment:
why does the version enforcement not work once java.version is set to 11?
```
          <execution>
            <id>enforce-versions</id>
            <goals>
              <goal>enforce</goal>
            </goals>
            <configuration>
              <rules>
                <requireMavenVersion>
                  <version>${maven.version}</version>
                </requireMavenVersion>
                <requireJavaVersion>
                  <version>${java.version}</version>
                </requireJavaVersion>
              </rules>
            </configuration>
          </execution>
```
does it need to be overridden in a different way? using this skip variable for java 11 seems hacky

Reply:
The current java version is hard coded in the enforce-version
```
<requireJavaVersion>
                  <version>[1.8, 1.9)</version>
                </requireJavaVersion>
```
I do not know whether your way of defining the version range work or not